vpr/src/base/read_options.cpp

```diff
@@ -1887,6 +1922,15 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("off")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument<e_check_route_option, ParseCheckRoute>(args.check_route, "--check_route")
+        .help(
+            "Options to run check route in three different modes.\n"
+            " * off    : check route is completely disabled.\n"
+            " * quick  : runs check route with slow checks disabled.\n"
+            " * full   : runs the full check route step.\n")
+        .default_value("quick")
```

Comment:
I'd lean towards keeping the default for this option as `full`. That ensures we do the most thorough checking by default (which is what we want, for example in regular VTR). Particularly if #1289 narrows the run-time difference between `quick` and `full`.

If someone (e.g. symbiflow) want to use quick, I feel that should be an end-user/flow override.

Reply:
Sure, moved the option default back to full